content/applications/finance/accounting/reporting/year_end.rst

```diff
@@ -2,115 +2,112 @@
+Closing a fiscal year
+---------------------
 
-- Run the optional **Payments Matching** feature, under the **More**
-  dropdown on the dashboard, validating any open **Vendor Bills** and
-  **Customer Invoices** with their payments. This step is optional,
-  however it may assist the year-end process if all outstanding
-  payments and invoices are reconciled, and could lead finding
-  errors or mistakes in the system.
+Then, to close the fiscal year:
 
-- Your accountant/bookkeeper will likely verify your balance sheet
-  items and book entries for:
+- Run a :ref:`tax report <reporting/tax-report>`, and verify that all tax information is correct.
+- Reconcile all accounts on the :ref:`balance sheet <reporting/balance-sheet>`:
 
-  - Year-end manual adjustments, using the **Adviser Journal Entries**
-    menu (For example, the **Current Year Earnings** and **Retained
-    Earnings** reports).
+  - Update the bank balances in Odoo according to the actual balances found on the bank statements.
+  - Reconcile all transactions in the cash and bank accounts by running the :ref:`aged receivables
+    <reporting/aged-receivable>` and :ref:`aged payables <reporting/aged-payable>` reports.
+  - Audit all accounts, being sure to fully understand all transactions and their nature, making
+    sure to include loans and fixed assets.
+  - Optionally, run :ref:`payments matching <payments/matching>` to validate any open vendor bills
+    and customer invoices with their payments. While this step is optional, it could assist the
+    year-end closing process if all outstanding payments and invoices are reconciled, potentially
+    finding errors or mistakes in the system.
 
-  - **Work in Progress**.
+Accountant verifications
+------------------------
 
-  - **Depreciation Journal Entries**.
+Next, the accountant/bookkeeper will likely verify balance sheet items and book entries for:
 
-  - **Loans**.
+  - year-end manual adjustments,
+  - work in progress,
+  - depreciation journal entries,
+  - loans, and
+  - tax adjustments.
 
-  - **Tax adjustments**.
+If the accountant/bookkeeper is going through the year-end audit, they may want to have paper copies
+of all balance sheet items (such as loans, bank accounts, prepayments, sales tax statements, etc.)
+to compare these with the balances in Odoo.
 
-If your accountant/bookkeeper is going through end of the year auditing,
-they may want to have paper copies of all balance sheet items (such as
-loans, bank accounts, prepayments, sales tax statements, etc...) to
-agree these against your Odoo balances.
+.. tip::
+   During this process, it is good practice to set a :guilabel:`Journal Entries Lock Date` to the
+   last day (inclusive) of the preceding fiscal year by going to :menuselection:`Accounting -->
+   Accounting --> Lock Dates`. This way, the accountant can be confident that nobody is changing the
+   transactions while auditing the books. *Accountant* users can still create and modify entries.
```

Comment:
"users with the accountant access rights"

Reply:
I would say: "Users from the _accountant_ access group"
NB: On 15.0, it's the _advisor_ access group. To modify from 16.0 onwards